sites/friday/src/get-started/express/add-database.md

```diff
@@ -0,0 +1,110 @@
+---
+title: "Add a database"
+weight: -130
+description: |
+  Once Your Express application has been deployed on {{% vendor/name %}}, you might want to add a service to your application.
+---
+
+{{% description %}}
+
+{{% vendor/name %}} projects already include a [variety of services](/add-services.html#available-services) so you don’t have to subscribe to an external cache or search-engine services.
+And as these services are included in your project, you can manage them through Git and they’re backed up along with the rest of your project.
+Your project source code defines the services configuration in the main `.{{% vendor/cli %}}/config.yaml` file and this is where you can add new services.
+
+As an example on how to do so, to add a [MariaDB database engine](/add-services/mysql.html) into your Express project, complete the following 4 steps:
+
+## Create a new branch
+
+## Add a MariaDB service
+Configure the MariaDB service by adding this Yaml definition at the end of your `.{{% vendor/cli %}}/config.yaml` file:
+
+```yaml
+# .{{% vendor/cli %}}/config.yaml
+{{< code-link destination="/add-services.html#available-services" text="services" title="Click to see the complete list of all available services" >}}:
+  db:
+    type: mariadb:10.4
+```
```

Comment:
This adds the little location tab to the top of the snippet.
```suggestion
```yaml {configFile="app"}
{{< code-link destination="/add-services.html#available-services" text="services" title="Click to see the complete list of all available services" >}}:
  db:
    type: mariadb:10.4
```
```